### PR TITLE
update the --skip-verify flag in the deployment page 

### PR DESCRIPTION
**Pull request by punitmundra**
### :white_check_mark: Checklist

**All PRs** must tick these:

- [ ] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [ ] All commits signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

With occasional exceptions, all PRs **from Progress employees** must tick these:

- [ ] Is the code clear? *(complicated code or lots of comments--subdivide and use well-named methods, meaningful variable names, etc.)*
- [ ] Consistency checked? *(user notifications, user prompts, visual patterns, code patterns, variable names)*
- [ ] Repeated code blocks eliminated? *(adapt and reuse existing components, blocks, functions, etc.)*
- [ ] Spelling, grammar, typos checked? *(at a minimum use `make spell` in any component directory)*
- [ ] Code well-formatted? *(indents, line breaks, etc. improve rather than hinder readability)*

All PRs **from Progress employees** should tick these if appropriate:

- [ ] Tests added/updated? (all new code needs new tests)
- [ ] Docs added/updated? (all customer-facing changes)

*Please add a note next to any checkbox above if you are NOT ticking it.*

### CODE REVIEW

File: components/docs-chef-io/content/automate/ha_onprim_deployment_procedure.md [hunk 1]
```diff
@@ -88,26 +88,24 @@ You can also view the [Sample Config](#sample-config).
...
     chef-automate deploy config.toml --airgap-bundle automate.aib
-    "
+    ```
+
+   If we want to skip verficiation in the deployment command we can use `--skip-verify` flag
+    ```bash
+     chef-automate deploy config.toml --airgap-bundle automate.aib --skip-verify
     ```
 
 1. Verify the deployment by checking status summary
```
Comment (bvtejaswi) [inline components/docs-chef-io/content/automate/ha_onprim_deployment_procedure.md:102]: Verify the deployment by checking status and info

File: components/docs-chef-io/content/automate/ha_onprim_deployment_with_aws_managed_deployment.md [hunk 2]
```diff
@@ -49,27 +49,26 @@ You can also view the [Sample Config](#sample-config-to-setup-on-premise-deploym
...
 ```
 
+If we want to skip verficiation in the deployment command we can use `--skip-verify` flag
+
+```bash
+ chef-automate deploy config.toml --airgap-bundle automate.aib --skip-verify
+```
 ## Verify Deployment
 
 Verify the deployment by checking status summary:
```
Comment (bvtejaswi) [inline components/docs-chef-io/content/automate/ha_onprim_deployment_with_aws_managed_deployment.md:65]: Verify the deployment by checking status and info

File: components/docs-chef-io/content/automate/ha_onprim_deployment_with_customer_managed_deployment.md [hunk 3]
```diff
@@ -49,27 +49,26 @@ See the steps [here](/automate/ha_onprim_deployment_procedure/#steps-to-run-on-b
...
+```
+
+If we want to skip verficiation in the deployment command we can use `--skip-verify` flag
+```bash
+  chef-automate deploy config.toml --airgap-bundle automate.aib --skip-verify
 ```
 
 ## Verify Deployment
 
 Verify the deployment by checking status summary:
```
Comment (bvtejaswi) [inline components/docs-chef-io/content/automate/ha_onprim_deployment_with_customer_managed_deployment.md:65]: Verify the deployment by checking status and info

File: components/docs-chef-io/content/automate/ha_onprim_deployment_procedure.md (hunk 1 above)
Comment (punitmundra) [reply components/docs-chef-io/content/automate/ha_onprim_deployment_procedure.md:102]: Why not the other 2 command

Comment (bvtejaswi) [reply components/docs-chef-io/content/automate/ha_onprim_deployment_procedure.md:102]: Verify the deployment by running following commands

**Review by bvtejaswi — APPROVED** [review]

**Pull request merged**